log.go

```diff
@@ -0,0 +1,58 @@
+package go2sky
+
+
+import (
+	"context"
+	"errors"
+)
+
+type ReportedLogData interface {
+	Context() context.Context
+	ErrorLevel() string
+	Data() string
+}
+
+type DefaultLogData struct {
+	LogCtx context.Context
+	LogErrLevel string
```

Comment:
```go
type LogLevel string

const (
	LogLevelDebug LogLevel = "debug"
	LogLevelInfo  LogLevel = "info"
	LogLevelWarn  LogLevel = "warn"
	LogLevelError LogLevel = "error"
)
```
I think should define constants for log level

Reply:
fixed